lib/msal-browser/docs/signed-http-request.md

```diff
@@ -0,0 +1,70 @@
+# Signed Http Request
+
+MSAL.js provides the `SignedHttpRequest` class as a convenience to assist in creating [signed http requests (SHRs)](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-signed-http-request-03) for payloads (e.g. tokens) that are acquired out-of-band from MSAL.js. This enables applications to leverage the same encryption and caching MSAL uses for [access token proof-of-possession](./access-token-proof-of-possession.md) for their own payloads.
+
+Note: The `SignedHttpRequest` class is intended for advanced use cases where the built-in access token proof-of-possession functionality cannot be used.
```

Comment:
How would I know I have an advanced use case that requires this?

Reply:
Will provide examples (e.g. using msal pop token signing for payload access tokens, or AAD access tokens acquired out of band from msal.